src/index.html

```diff
@@ -7,6 +7,344 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <a href="#logo" class="header__logo">
+        <img src="images/logo.png" alt="logo">
+      </a>
+      <nav class="nav">
+        <ul class="nav__list">
+          <li>
+            <a class="nav__link is-active" href="#apple">Apple
+            </a>
+          </li>
+          <li>
+            <a class="nav__link" href="#samsung">Samsung
+            </a>
```

Comment:
```suggestion
            <a class="nav__link" href="#samsung">
              Samsung
            </a>
```

check your code style
fix all such cases